### ISSUE TITLE
spinner 3000

### ISSUE THREAD

**Issue by lottiecoxon** [PostHog/posthog#15569]
# Spinner 3000 
### Why?
 - old spinner doesn't work on new darkmode + looks old in general 
### When? 
- Now -ish 
### What? 
- I have a few ideas below 
### DISCLAIMER
All these are drafts /frames to show general ideas and not final pieces as it takes a while to add colour and full movement. 

### Balance Logo 
An old concept that never got picked up, but maybe now is it's time to shine? I like how it's playful yet incorporates the logo. Downsides it isn't very descriptive of what we do as a company but tbh that wouldn't be the end of the world...

https://github.com/PostHog/posthog/assets/65415371/4bb29f22-badb-4225-9b3f-c76c92533876

### Vintage dial up internet with pixel hog 
Taking inspiration from old dial up imagrey, a nice throw back as it were

https://github.com/PostHog/posthog/assets/65415371/57dc718b-b490-45d3-a116-fb6a066ce172

### Button/Screen swipe 
This concept would take inspiration from a [button animation for notion](https://dribbble.com/shots/15105524-Notion-Logo-Animation-Concept) that I saw. You press the PostHog button, and then it brings up 3 tabs that get swiped through, back to button - repeat. 

I think this could look really cool, I just would need to spend the time on adding in dynamic motion.

https://github.com/PostHog/posthog/assets/65415371/fef0ac20-ade4-49ef-afd6-4159311eb58a

### Key features displayed in animation 
Does what it says on the tin, feature flag buttons, opening graphs, A/B test sliders etc

https://github.com/PostHog/posthog/assets/65415371/0b3ddfbe-52c8-44fa-86b8-5a747bdbfa02

**Comment by jamesefhawkins** [comment]
i think the colorful version draws too much attention to itself, don't feel strongly about the rest other than i think they're cool!

**Comment by lottiecoxon** [comment]
Yeah, happy accident tbh with the lack of colour for the rest of them but I think it works quite well the more I look at them.
I think either the last one or the one with the internet dial up would work the best

**Comment by corywatilo** [comment]
These are all cute. But given 3000 is about being slick/clean/simple, I sort of wonder if less is more? (As in, something that doesn't draw attention to itself - just something more standard.) Will think on this a bit...

**Comment by lottiecoxon** [comment]
I tried out an idea of a spinner box with deconstructed logo parts falling within it, but unsure if it's tooooo conceptually weird for a spinner 

https://github.com/PostHog/posthog/assets/65415371/d8da3032-4fdb-41a8-89a9-df900be81809

https://github.com/PostHog/posthog/assets/65415371/470605a0-00d0-42a9-a618-00d3baa75b88

**Comment by Twixes** [comment]
TBH I quite liked the previous spinner we had, it was simple but it had this _fluid_ quality to it.
<img src="https://github.com/PostHog/posthog/assets/4550621/ea43045b-51b6-41ff-8e6e-336da3770538" alt="Spinner" width="134" />

**Comment by lottiecoxon** [comment]
last try to spark joy - 

https://github.com/PostHog/posthog/assets/65415371/530981a6-ce7d-4295-907a-a7a648c9b08e

**Comment by lottiecoxon** [comment]
Concept - similar to a spinner but with more code flare to it 

https://github.com/PostHog/posthog/assets/65415371/c71fa5b9-3c9b-4124-a388-304ff2aae82c

https://github.com/PostHog/posthog/assets/65415371/36cd19d0-3c77-4e67-a2a9-4aa4feed6e7e

**Comment by lottiecoxon** [comment]
I added in a gradient to the option above and I think this could be a strong contender 

@Twixes thoughts on this?

https://github.com/PostHog/posthog/assets/65415371/cd082b44-77f8-4f7a-a469-e6b62db169d2

**Comment by Twixes** [comment]
This is pretty cool as a larger loading indicator. Not sure about the legibility if we used this within something like a button – would love an example of that.
I think the gradient might be a bit too strong as is, but it looks cool. I wonder if this sort of thing could fit into our icons @corywatilo? Or whether we're going for a totally flat look.

**Comment by corywatilo** [comment]
I think it'd be cool to use some animated spinner for special actions like:

1. Upgrading a plan
2. On sign in

This way they still spark joy but don't start to feel tired from constant repetition.

So when navigating between pages of the app or filtering/sorting data, minimal is better as it will be slicker. (This would be some combination of an animated horizontal line or circle, and skeleton states.)

**Comment by corywatilo** [comment]
@lottiecoxon One area where could definitely have some fun is in the session replay loading state. Right now it just looks like this:

<img width="1389" alt="image" src="https://github.com/PostHog/posthog/assets/154479/bf87d476-7984-42de-b17a-576c95f8c332">

Since replays _do_ take some time to load, having something nice here would be a huge lift IMO.

cc @benjackwhite

**Comment by corywatilo** [comment]
The loading state we currently have uses our colors but isn't super brandable.

<img width="508" alt="image" src="https://github.com/PostHog/posthog/assets/154479/fb45eaa5-2c3f-47c7-8619-7e65362b1b17">

Instead, what if we animated our logo (like a hedgehog grows from nothing)? Like each individual piece spawns from nothing.

<img width="488" alt="image" src="https://github.com/PostHog/posthog/assets/154479/5a083b63-3ce6-41b2-9e21-f822929b1aba">

**Comment by lottiecoxon** [comment]
I drew up some concepts for the buffering page below : 
(personal favourite is the 'hitting the problem away' one

![Untitled_Artwork 365](https://github.com/PostHog/posthog/assets/65415371/5351fdd2-99e4-4025-a9cb-97f5ab510597)

![Untitled_Artwork 363](https://github.com/PostHog/posthog/assets/65415371/92c8f0da-c047-400f-815c-dd0bc8ef4bf3)

![Untitled_Artwork 364](https://github.com/PostHog/posthog/assets/65415371/c2fc79b9-e085-4218-bfae-c45190a46d58)

**Comment by lottiecoxon** [comment]
I also looked into trying to make the code influenced spinner work better and chunked out the lines so it's more visible at smaller sizes + as well as making it in our signature blue colour - is this worth testing in the app format to see if it works with text? 
It's currently within the rive app (as I tested them out vs lottiefiles but can easily make it again in lottie if needed) 

https://github.com/PostHog/posthog/assets/65415371/f9f535ec-decd-4b99-9802-226bd1341f0a

**Comment by lottiecoxon** [comment]
I think this is fun (rough storyboard for now) but maybe the hammer is a bit too aggressive ?

 
![Untitled_Artwork 27](https://github.com/PostHog/posthog/assets/65415371/0a221784-8dff-44be-b592-956080db6480)
